benchmark/src/main.rs

```diff
@@ -135,7 +143,7 @@ fn main() {
         let (r, w) = evmap::Options::default()
             .with_capacity(5_000_000)
             .construct();
-        let w = sync::Arc::new(sync::Mutex::new((w, 0, refresh)));
+        let w = sync::Arc::new(parking_lot::Mutex::new((w, 0, refresh)));
```

Comment:
I wonder if `evmap` should provide this kind of shared write handle as a part of its API... What do you think?

Reply:
I think the read/write handles make sense as-is, and there may even be situations where a mutex is unnecessary for the writer, e.g. some event loop that owns the writer and can use it mutably without a lock.

Reply:
I was more thinking that we would provide this wrapper _in addition_ to the non-`Mutex` one so people don't have to do the wrapping themselves.

Reply:
If we're benchmarking evmap with `parking_lot`, we should also change the benchmark of the standard library hashmap to use `parking_lot`.

Reply:
I thought about this, and I wanted the *least* overhead for the writers themselves. `parking_lot` is still faster than `std`, and we didn't want to benchmark the writer lock, only the concurrent map. However, for the `std` benchmark, the lock is part of that concurrent map, so it stayed on `std::sync::Mutex`.

I can revert this as well if you want.

Reply:
I'd argue that `parking_lot` *is* part of the concurrent map for `evmap` if you indeed want to have multiple write handles, so I think `std` should also get the advantage of using `parking_lot`. It won't materially change the results, but it is a fairer comparison.